lib/vega_lite/export.ex

```diff
@@ -39,13 +39,13 @@ defmodule VegaLite.Export do
           to_html(vl)
 
         :png ->
-          to_png(vl)
+          to_png(vl, Keyword.delete(opts, :format))
```

Comment:
Instead of deleting it here, you can change get_lazy a couple lines above to do this:

```
{format, opts} =
  Keyword.pop_lazy(...
```

:)